contrib_bots/lib/define_bot.py

```diff
@@ -0,0 +1,81 @@
+# See readme.md for instructions on running this code.
+import logging
+import json
+import requests
+
+class DefineHandler(object):
+    '''
+    This plugin define a word that the user inputs. It
+    looks for messages starting with '@define'.
+    '''
+
+    DEFINITION_API_URL = 'https://owlbot.info/api/v1/dictionary/{}?format=json'
+    REQUEST_ERROR_MESSAGE = 'Definition not available.'
+    EMPTY_WORD_REQUEST_ERROR_MESSAGE = 'Please enter a word to define.'            
```

Comment:
Slightly weird variable name, also it is used by both more than 1 word and no word.